packages/superset-ui-plugin-chart-table/src/Table.tsx

```diff
@@ -71,7 +72,45 @@ function getText(value: unknown) {
   return String(value);
 }
 
+type columnWidthMetaDataType = {
+  [key: string]: {
+    maxWidth: number;
+    width: number;
+  };
+};
+
 class TableVis extends React.PureComponent<InternalTableProps, TableState> {
+  columnWidthSelector = createSelector<
+    ParentRow[],
```

Comment:
Are the generic necessary? Is it possible to use the same way with other selectors that only the first argument type is defined?